app/views/users/show.html.erb

```diff
@@ -1,5 +1,32 @@
-<h2>User Details</h2>
-<p>Name: <%= @user.name %></p>
-<p>Email: <%= @user.email %></p>
-<p>Bio: <%= @user.bio %></p>
-<p>Total Posts: <%= @user.posts_counter %></p>
+<div class="user-profile-container">
+  <div class="user-profile">
+    <div class="user-image">
+      <%= image_tag @user.photo, alt: @user.name, class: "profile-photo" %>
+    </div>
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+      <div class="user-bio">
+        <h3>Bio</h3>
+        <p><%= @user.bio %></p>
+      </div>
+    </div>
+  </div>
+
+  <div class="user-posts">
+    <h3>Posts:</h3>
+    <% @user.posts.each do |post| %>
```

Comment:
* According to the wireframe you are to have a method that returns 3 most recent posts of the user so in your user show view file you will render the posts like this `<% @user.recent_post.each do |post| %>` 
![image](https://github.com/cancelei/blog-in-rails/assets/52497246/8bb65613-8975-4cae-830a-35f1554b7ec6)